TwoPageEBookReader/README.md

```diff
@@ -0,0 +1,22 @@
+# eBook Reader sample using Window Manager
+
+This sample demonstrates the Two Page design pattern for foldable devices. In this pattern, apps can take advantage of a foldable device's natural boundary to show multiple items from a collection. EBooks are a perfect application of this pattern, as their physical predecessors could be called "dual-leaf hinged devices". See other design patterns in [Microsoft's UX Design documentation](https://docs.microsoft.com/en-us/dual-screen/introduction).
```

Comment:
Remove `/en-us` from the URL - we generally do this whenever we share docs links, so the reader gets redirected to their own language if <> English

Reply:
done